packages/definitions-parser/src/git.ts

```diff
@@ -66,62 +56,74 @@ export async function gitDiff(log: Logger, definitelyTypedPath: string): Promise
     return stdout;
   }
 }
-
-/** Returns all immediate subdirectories of the root directory that have changed. */
-export function gitChanges(diffs: GitDiff[]): PackageId[] {
-  const changedPackages = new Map<string, Map<string, DependencyVersion>>();
-
+/** Returns all immediate subdirectories of the root directory that have been deleted. */
+function gitDeletions(diffs: GitDiff[]): { errors: string[] } | { ok: PackageId[] } {
+  const changedPackages = new Map<string, PackageId>();
+  const errors = [];
   for (const diff of diffs) {
+    if (diff.status !== "D") continue;
```

Comment:
It will test things that might not need to be tested (but since they're complicated enough that git didn't understand, probably *should* be tested.) That's all that gitDeletions is used for now.